TAF/Tests/SearchTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using TAF.Core.Browser;
+using TAF.Core.Elements;
+using TAF.Core.Utilities;
+using TAF.Web.Pages;
+
+namespace TAF.Tests
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    public class SearchTests : BaseTest
+    {
+        private MainPage _mainPage;
+        private SearchResultsPage _searchResultsPage;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mainPage = new();
+            _searchResultsPage = new();
+
+            _mainPage.AcceptAllCookies();
```

Comment:
It is better to move accept cookies to the BaseTest as you use it in several classes.